index.yaml

```diff
@@ -101,6 +101,13 @@ indexes:
   - name: last_updated
     direction: desc
 
+- kind: FeedbackMessageModel
```

Comment:
I tried everything but couldn't back this. I ran all tests too. I don't know really know what to do. :(